python/lsst/ts/xml/data/sal_interfaces/LEDProjector/LEDProjector_Commands.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+  <SALCommand>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_command_turnAllLEDsOn</EFDB_Topic>
+    <Description>Turn on all LEDs connected to the labjack.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_command_turnAllLEDsOff</EFDB_Topic>
+    <Description>Turn off all LEDs connected to the labjack.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_command_turnOnLED</EFDB_Topic>
```

Comment:
I'd suggest following the same pattern you used for the previous 2 commands. Consider renaming this `turnLEDOn`, or you could drop "LED" and call it `turnOn`. However, and being extra picky here, consider using `switch` instead  of `turn`.